nautobot/virtualization/models.py

```diff
@@ -273,10 +273,10 @@ def clean(self):
             ip = getattr(self, field)
             if ip is not None:
                 if field == "primary_ip4":
-                    if ip.family != 4:
+                    if ip.ip_version != 4:
```

Comment:
Not blocking for this issue, but I noticed via grep that the IPAM querysets still have an `ip_family` method, wonder if we should be renaming that to `version` or `ip_version`? Or maybe actually removing it altogether as unnecessary now that `version` is a concrete field?